scripts/nymms_api

```diff
@@ -10,4 +12,8 @@ logger = cli.setup_logging(args.verbose)
 config.load_config(args.config)
 
 from nymms.api import routes
+
+for plugin in config.settings.get('api', []).get('plugins', []):
```

Comment:
Well, I didn't really know how to solve that problem.  I figure this way isn't too unlike adding urls of django plugins for instance.  In both cases, you could end up in an invalid state.

It could also be addressed by docs (handler specific).